source/plugins/go/src/go.mod

```diff
@@ -1,35 +1,88 @@
 module Docker-Provider/source/plugins/go/src
 
-go 1.14
+go 1.19
```

Comment:
i remembered, we have observed fluent-bit crashing when we upgraded the go version. did we verify that this upgrade works as expected without any issues?

Reply:
that was with go 1.20 or go 1.21 but not with go 1.19